ui/src/app/views/workspace-edit/component.html

```diff
@@ -14,11 +14,28 @@ <h2 class="page-header">Clone workspace "{{oldWorkspaceName}}"</h2>
       </ng-container>
       <div class="required-text">(Required)</div>
     </div>
-    <div class="name-area required-field" [class.validation-error]="nameNotEntered">
+    <div class="name-area required-field" [class.validation-error]="nameNotEntered"
+          style="display: inline-block;">
       <input type="text" class="input name" name="workspace-name" autofocus
              [(ngModel)]="workspace.name" [disabled]="!hasPermission"
              placeholder="Workspace Name">
     </div>
+    <clr-dropdown style="display: inline-block;">
+      <button [disabled]="mode == Mode.Edit"
```

Comment:
I think it would be good to have a tooltip to explain why you can't edit it.

Reply:
Done